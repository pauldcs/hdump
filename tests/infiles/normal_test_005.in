assets/AAAA --size 16