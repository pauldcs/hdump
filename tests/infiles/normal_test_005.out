0000000+0  41 41 41 41 41 41 41 41  41 41 41 41 41 41 41 41  AAAAAAAAAAAAAAAA
